### PR TITLE
feat: app specific directory storage

### LINKED ISSUE #544: Shift to App Specific Directory for storing Badge and Clipart's
**Describe the bug**
We are currently using the external storage to save badges and clipart's. Lets change the directory to app specific external storage.

**Additional context**
Change the current directory from External Storage to App Specific directory.

### PR DESCRIPTION
**Pull request by refactor-droidyy**
Fixes #544 Shift to App-Specific Directory for storing Badge and Clipart's

### CODE REVIEW

**Comment by refactor-droidyy** [comment]
@adityastic please review.

**Comment by refactor-droidyy** [comment]
@adityastic changed one.

File: app/src/main/java/org/fossasia/badgemagic/util/StorageUtils.kt [hunk 1]
```diff
@@ -19,12 +19,12 @@ import org.fossasia.badgemagic.data.fragments.CONF_SPEED
 import org.fossasia.badgemagic.data.fragments.ConfigInfo
 import org.json.JSONObject
 
-object StorageUtils {
+class StorageUtils(val context: Context) {
     private val EXTERNAL_STORAGE_DIRECTORY = "${Environment.getExternalStorageDirectory()
```
Comment (adityastic) [inline app/src/main/java/org/fossasia/badgemagic/util/StorageUtils.kt]: try to change Environment.getExternalStorageDirectory to app specific directory code now

**Comment by refactor-droidyy** [comment]
@adityastic changed the Environment.getExternalStorageDirectory to app-specific directory code :)

**Comment by adityastic** [comment]
@refactor-droidyy yes, you need to pass the object in the async task

**Comment by refactor-droidyy** [comment]
@adityastic everything is working fine now.

**Comment by refactor-droidyy** [comment]
@adityastic once again thank you very much I didn't know much about injecting dependency managing singletons and its vast use. But now with this PR and your guidance I really learned a lot. :)

**Pull request merged**